tests/installation/add_serial_console.pm

```diff
@@ -33,8 +33,8 @@ sub run {
     assert_script_run('sed -ie \'s/GRUB_TERMINAL.*//\' /etc/default/grub');
     # VMware needs to always stop in Grub and wait for interaction.
     # Migration seems to reset it.
-    if (get_var('KEEP_GRUB_TIMEOUT')) {
-        assert_script_run('sed -ie \'s/GRUB_TIMEOUT.*/GRUB_TIMEOUT=8/\' /etc/default/grub');
+    if (get_var('GRUB_TIMEOUT')) {
+        assert_script_run('sed -ie \'s/GRUB_TIMEOUT.*/GRUB_TIMEOUT=' . get_var('GRUB_TIMEOUT') . '/\' /etc/default/grub');
```

Comment:
Allowing customized GRUB_TIMEOUT is great. But will deleting KEEP_GRUB_TIMEOUT be problematic for other squads' tests which are setting KEEP_GRUB_TIMEOUT only? Maybe some minor change with backward compatibility will survive all.

Reply:
Okay, if you are sure that our test is the only user, then it will be fine.